.github/workflows/wheels.yml

```diff
@@ -31,6 +31,14 @@ jobs:
       - name: Make sdist
         run: |
           pipx run build --sdist
+          
+      - name: Install from sdist
+        run: |
+          pip install --upgrade pip setuptools wheel
+          pip install dist/h3-*.tar.gz[all]
```

Comment:
I couldn't get it to work with both * and [all], hence why I first used the glob to rename the file

Reply:
ah, ok!